blocklogger.go

```diff
@@ -53,7 +53,7 @@ func (b *blockProgressLogger) logBlockHeight(block *dcrutil.Block) {
 	b.receivedLogTickets += int64(block.MsgBlock().Header.FreshStake)
 	now := time.Now()
 	duration := now.Sub(b.lastBlockLogTime)
-	if duration < time.Second*10 {
+	if (block.Height() < syncHeight) && (duration < time.Second*10) {
```

Comment:
it doesn't look like `gofmt -s` is being checked by the CI testing (which i actually agree with), but this can still be simplified and we probably should to remove the unnecessary parenthesis.

Reply:
`gofmt -s` doesn't actually identify that when I run it but agreed that the parenthesis are unnecessary so I removed them